manifests/client.pp

```diff
@@ -97,10 +97,12 @@
     notify  => Service[$agent_service_name],
   }
 
+  Concat::Fragment {
+    target => 'ossec.conf',
+    notify => Service[$agent_service_name]
+  }
+
```

Comment:
true that. 
I see only 2 alternatives: 
  * to include those attributes in all the specific files declarations
  * to create a hash with the defaults and then create the files with `create_resources` passing the hash as defaults (not sure if the references to require/notify would work in this case).